share/OpenMS/SCHEMAS/IdXML_1_4.xsd

```diff
@@ -249,7 +249,7 @@
 				<xs:documentation>significance threshold as calculated by the search engine</xs:documentation>
 			</xs:annotation>
 		</xs:attribute>
-		<xs:attribute name="spectrum_reference" type="xs:unsignedInt">
+                <xs:attribute name="spectrum_reference" type="xs:string">

```

Comment:
I am, in principle, fine with having the change if it was a bug in the idXML schema file. But you should definitely update the schema documentation (still refers to an integer)